test/terraform_scp_example_test.go

```diff
@@ -112,9 +112,11 @@ func testScpDirFromHost(t *testing.T, terraformOptions *terraform.Options, keyPa
 	// as we know the Instance is running an Ubuntu AMI that has such a user
 	sshUserName := "ubuntu"
 	publicHost := ssh.Host{
-		Hostname:    publicInstanceIP,
-		SshKeyPair:  keyPair.KeyPair,
-		SshUserName: sshUserName,
+		Hostname: publicInstanceIP,
+		Options: &ssh.Options{
```

Comment:
It looks like you forgot to update these.

Reply:
on it